### PR TITLE
Fix Night Mode Inactive Button Colour

### LINKED ISSUE #12127: Night Mode Inactive Button Colour
#### Issue Description

In Night Mode: whether the buttons in settings are active or inactive ("On" or "Off"), they are always the same colour, Green, which is somewhat misleading.  I tested on three different desktop browsers and all seem to be doing the same thing.

#### Browser Information

* Browser Name, Version: Chrome, 55.0.2883.95 (64-bit) | Safari, Version 10.0.1 | Firefox, 50.1.0
* Operating System: OS X El Capitan 10.11.6
* Desktop

#### Screenshot

screen shot 2016-12-20 at 9 21 11 am

### PR DESCRIPTION
**Pull request by MarkusToe**
#### Pre-Submission Checklist

- [x] Your pull request targets the `staging` branch of FreeCodeCamp.
- [x] Branch starts with either `fix/`, `feature/`, or `translate/` (e.g. `fix/signin-issue`)
- [x] You have only one commit (if not, [squash](http://forum.freecodecamp.com/t/how-to-squash-multiple-commits-into-one-with-git/13231) them into one commit).
- [x] All new and existing tests pass the command `npm test`. Use `git commit --amend` to amend any fixes.

#### Type of Change

- [x] Small bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds new functionality)
- [ ] Breaking change (fix or feature that would change existing functionality)
- [ ] Add new translation (feature adding new translations)

#### Checklist:

- [x] Tested changes locally.
- [x] Closes currently open issue (replace XXXX with an issue no): Closes #12127

#### Description
I added a new class `.btn-toggle` and set the background-color to transparent in night mode. Buttons with the classes `.btn-toggle` and `.active` will get the darkgreen background.

### CODE REVIEW

**Comment by raisedadead** [comment]
@MarkusToe could you squash your commits? Thanks.

File: common/app/routes/settings/components/Locked-Settings.jsx [hunk 1]
```diff
@@ -1,37 +1,38 @@
-import React, { PropTypes } from 'react';
-import { Button, Row, Col } from 'react-bootstrap';
+import React, {PropTypes} from 'react';
```
Comment (BerkeleyTrue) [inline common/app/routes/settings/components/Locked-Settings.jsx]: Re-add padding

File: common/app/routes/settings/components/Locked-Settings.jsx [hunk 2]
```diff
@@ -1,37 +1,38 @@
...
+import {Button, Row, Col} from 'react-bootstrap';
```
Comment (BerkeleyTrue) [inline common/app/routes/settings/components/Locked-Settings.jsx]: Re-add padding

File: common/app/routes/settings/components/Locked-Settings.jsx [hunk 3]
```diff
@@ -1,37 +1,38 @@
...
+                </Button>
+            </Col>
+        </Row>
+    );
 }
 
 LockSettings.propTypes = {
-  isLocked: PropTypes.bool,
-  toggle: PropTypes.func.isRequired
+    isLocked: PropTypes.bool,
```
Comment (BerkeleyTrue) [inline common/app/routes/settings/components/Locked-Settings.jsx]: Remove added space

File: common/app/routes/settings/components/Email-Setting.jsx [hunk 4]
```diff
@@ -92,7 +96,11 @@ export default function EmailSettings({
             bsSize='lg'
             bsStyle='primary'
             className={
-              classnames('positive-20', { active: sendNotificationEmail })
+              classnames(
+                  'positive-20',
+                  { active: sendNotificationEmail },
+                  'btn-toggle'
+              )
```
Comment (BerkeleyTrue) [inline common/app/routes/settings/components/Email-Setting.jsx]: Should be two space indent

File: common/app/routes/settings/components/Email-Setting.jsx [hunk 5]
```diff
@@ -70,7 +70,11 @@ export default function EmailSettings({
             bsSize='lg'
             bsStyle='primary'
             className={
-              classnames('positive-20', { active: sendMonthlyEmail })
+              classnames(
+                  'positive-20',
+                  { active: sendMonthlyEmail },
+                  'btn-toggle'
```
Comment (BerkeleyTrue) [inline common/app/routes/settings/components/Email-Setting.jsx]: Should be two space indent

File: common/app/routes/settings/components/Email-Setting.jsx [hunk 6]
```diff
@@ -114,7 +122,11 @@ export default function EmailSettings({
             bsSize='lg'
             bsStyle='primary'
             className={
-              classnames('positive-20', { active: sendQuincyEmail })
+              classnames(
+                  'positive-20',
+                  { active: sendQuincyEmail },
+                  'btn-toggle'
```
Comment (BerkeleyTrue) [inline common/app/routes/settings/components/Email-Setting.jsx]: Should be two space indent

File: common/app/routes/settings/components/Locked-Settings.jsx [hunk 7]
```diff
@@ -1,37 +1,38 @@
...
-          { isLocked ? 'On' : 'Off' }
-        </Button>
-      </Col>
-    </Row>
-  );
+export default function LockSettings({isLocked, toggle}) {
+    const className = classnames({
+        'positive-20': true,
+        active: isLocked,
+        'btn-toggle': true
```
Comment (BerkeleyTrue) [inline common/app/routes/settings/components/Locked-Settings.jsx]: Should be two space indent

File: common/app/routes/settings/components/Locked-Settings.jsx [hunk 8]
```diff
@@ -1,37 +1,38 @@
...
-          className={ className }
-          onClick={ toggle }
-          >
-          { isLocked ? 'On' : 'Off' }
-        </Button>
-      </Col>
-    </Row>
-  );
+export default function LockSettings({isLocked, toggle}) {
+    const className = classnames({
```
Comment (BerkeleyTrue) [inline common/app/routes/settings/components/Locked-Settings.jsx]: Should be two space indent

File: common/app/routes/settings/components/Locked-Settings.jsx [hunk 9]
```diff
@@ -1,37 +1,38 @@
...
+                    block={ true }
+                    bsSize='lg'
+                    bsStyle='primary'
+                    className={ className }
+                    onClick={ toggle }
+                    >
+                    { isLocked ? 'On' : 'Off' }
+                </Button>
+            </Col>
+        </Row>
```
Comment (BerkeleyTrue) [inline common/app/routes/settings/components/Locked-Settings.jsx]: Should be two space indent

**Review by BerkeleyTrue — CHANGES_REQUESTED** [review]
Indenting should always be two spaces.

**Comment by raisedadead** [comment]
The OP seems to be away. I have addressed the comments, please check and review.

File: common/app/routes/settings/components/Locked-Settings.jsx [hunk 10]
```diff
@@ -2,10 +2,11 @@ import React, { PropTypes } from 'react';
 import { Button, Row, Col } from 'react-bootstrap';
 import classnames from 'classnames';
 
-export default function LockSettings({ isLocked, toggle }) {
+export default function LockSettings({isLocked, toggle}) {
```
Comment (BerkeleyTrue) [inline common/app/routes/settings/components/Locked-Settings.jsx]: Re-add padding

**Comment by raisedadead** [comment]
@BerkeleyTrue addressed.

**Review by BerkeleyTrue — APPROVED** [review]

**Comment by BerkeleyTrue** [comment]
Code LGTM

**Comment by dhcodes** [comment]
@raisedadead is this good to go?

**Comment by raisedadead** [comment]
Yes, I have fixed all changes, this has been sitting here for a merge, can you please check and merge?

**Comment by dhcodes** [comment]
Looks good @raisedadead but while this is open, what do you think about changing the `btn-primary` border-color to `#777`. The black border blends in almost completely with the dark gray background. 
Here's the line:  https://github.com/freeCodeCamp/freeCodeCamp/blob/0a6024769bb7d7c0625309ab23f9d6a25c7a8064/client/less/main.less#L1132

Here's a screenshot: nightbutton

**Comment by raisedadead** [comment]
@dhcodes this is ready for a review once more 😉 !

**Comment by dhcodes** [comment]
magnifique! 👍  Thanks for putting up with me @raisedadead. 🌮  💯

**Pull request merged**